client/src/components/all-components/SiteAccessibilityButton.js

```diff
@@ -0,0 +1,142 @@
+import React, { useState } from "react";
+import { bindActionCreators } from "redux";
+import { connect } from "react-redux";
+import { actions as appActions } from "../../reducers/lab1/AppReducer";
+import { actions as mainActions } from "../../reducers/MainReducer";
+import { Panel as ColorPickerPanel } from "rc-color-picker";
+import AccessibilityImage from "../../assets/images/accessibility_icon.png";
+import { Collapse, Card, CardHeader, CardBody } from "reactstrap";
+import {
+  changeTSize,
+  setTextColor,
+  setBackgroundColor,
+} from "../footer/edit/editPage";
+
+const mapStateToProps = (state) => {
+  return {
+    // General
+    state: state,
+  };
+};
+
+const mapDispatchToProps = (dispatch) => {
+  return {
+    actions: bindActionCreators({ ...appActions, ...mainActions }, dispatch),
+  };
+};
+
+const SiteAccessibilityButton = () => {
```

Comment:
It seems like the button is not evenly positioned

![Screenshot 2024-11-20 at 2 10 30 PM](https://github.com/user-attachments/assets/5903d909-f73f-4798-9d73-14e6732cc8f7)

Reply:
It looks like the button still needs to be positioned evenly

![Screenshot 2024-11-22 at 1 48 08 PM](https://github.com/user-attachments/assets/54c30eac-cb6d-4fd0-99be-3f761f4fa09a)